manifests/charts/istio-control/istio-discovery/files/waypoint.yaml

```diff
@@ -44,19 +46,21 @@ spec:
           (omit .InfrastructureAnnotations "kubectl.kubernetes.io/last-applied-configuration" "gateway.istio.io/name-override" "gateway.istio.io/service-account" "gateway.istio.io/controller-version")
           (strdict "istio.io/rev" (.Revision | default "default"))
           (strdict
-            "ambient.istio.io/redirection" "disabled"
+            "ambient.istio.io/redirection" $redirection
             "prometheus.io/path" "/stats/prometheus"
             "prometheus.io/port" "15020"
             "prometheus.io/scrape" "true"
           ) | nindent 8 }}
       labels:
+        {{- $TunnelLabel := or (and (not $sandwich) (strdict "networking.istio.io/tunnel" "http")) (strdict) }}
```

Comment:
https://docs.google.com/document/d/1DAM_6SajCyC2hD8wBUraaZWZJqUVYPyemzw0-xpHmdY/edit?disco=AAABGF3VdDw

This can eventually be derived from the gateway listeners rather than redirection status